src/pages/Forms.mdx

```diff
@@ -150,6 +150,52 @@ import Icon from "../core/Icon";
       </label>
     </div>
   </div>
+  <div className="mb-40">
+    <h4 className="ui-text-h3 mb-16">Disabled checkboxes</h4>
+    <div className="ui-checkbox-p2">
+      <input
+        data-ui-checkbox-native=""
+        type="checkbox"
+        id="checkbox-example-4"
```

Comment:
We'll need to use unique ID's here, as we're currently interacting with the previous checkboxes in Storybook when clicking these labels